pandas/tests/plotting/test_converter.py

```diff
@@ -373,3 +373,21 @@ def test_convert_nested(self):
         r1 = self.pc.convert([data, data], None, self.axis)
         r2 = [self.pc.convert(data, None, self.axis) for _ in range(2)]
         assert r1 == r2
+
+
+class TestTimeDeltaConverter:
+    """Test timedelta converter"""
+
+    @pytest.mark.parametrize(
+        "x, decimal, format_expected",
+        [
+            (0.0, 0, "00:00:00"),
+            (3972320000000, 1, "01:06:12.3"),
+            (713233432000000, 2, "8 days 06:07:13.43"),
+            (32423432000000, 4, "09:00:23.4320"),
+        ],
+    )
+    def test_format_timedelta_ticks(self, x, decimal, format_expected):
+        tdc = converter.TimeSeries_TimedeltaFormatter
+        result = tdc.format_timedelta_ticks(x, pos=None, n_decimals=decimal)
```

Comment:
would it be worthwhile to parametrize over `pos` values here?  or would that obscure the logic we actually care about testing?

Reply:
pos is not being used in `format_timdelta_ticks` function, honestly i am not sure why/how this is used. So i just keep as is for now. @jbrockmendel

Reply:
Leaving it as is sounds fine, thanks for looking into it.